lib/rspecq/queue.rb

```diff
@@ -274,7 +281,10 @@ def job_build_timing(job)
     # Persist build timings to the global timings key, so that they can be
     # used for scheduling future builds.
     def update_global_timings(dst = key_timings)
+      dst_load = dst.sub(key_timings, key_load_timings)
```

Comment:
This is a hack, we should probably rethink the arguments for `update_global_timings`, perhaps we could just pass a `prefix` argument?